zkvm/src/txlog.rs

```diff
@@ -31,6 +32,27 @@ pub struct TxID(pub [u8; 32]);
 #[derive(Copy, Clone, PartialEq, Debug)]
 pub struct UTXO(pub [u8; 32]);
 
+/// MerkleHash represents a step in a Merkle proof of inclusion.
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub enum MerkleHash {
+    Left([u8; 32]),
+    Right([u8; 32]),
+}
+
+/// MerkleRoot defines the root of a Merkle tree.
+pub struct MerkleRoot {
```

Comment:
Maybe like this? So we can say "MerkleTree.root" instead of "MerkleRoot.root".
```suggestion
pub struct MerkleTree {
```